### PR TITLE
mcarr/Trading Partners

### PR DESCRIPTION
**Pull request by MikeQCarr**
### **What:**
- Add trading partners models to query the Trading Partner API
### **Why:**
- Search for trading partners

### CODE REVIEW

**Comment by alexander-clark** [comment]
might want to add .DS_Store to .gitignore. Otherwise looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.10.0] - [2020-04-08]
```
Comment (mmcrockett) [inline CHANGELOG.md:7]: Date today? `2020-04-09`

File: CHANGELOG.md [hunk 2]
```diff
@@ -62,7 +70,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Provider
 - Subscriber
 - Encounter
-- Eligibility 
+- Eligibility
 - Authentication
 - Configuration
```
Comment (mmcrockett) [inline CHANGELOG.md:76]: Sorry, I forgot to fix these in my release, do you mind adding `0.9.0` and yours too?

File: CHANGELOG.md (hunk 1 above)
Comment (MikeQCarr) [reply CHANGELOG.md]: @mmcrockett Updated!

File: CHANGELOG.md (hunk 2 above)
Comment (MikeQCarr) [reply CHANGELOG.md:76]: @mmcrockett Done!

**Pull request merged**